pkg/providers/aws/provider_redis.go

```diff
@@ -251,21 +255,26 @@ func getReplicationGroups(cacheSvc elasticacheiface.ElastiCacheAPI) ([]*elastica
 }
 
 // getRedisConfig retrieves the redis config from the cloud-resources-aws-strategies configmap
-func (p *AWSRedisProvider) getRedisConfig(ctx context.Context, r *v1alpha1.Redis) (*elasticache.CreateReplicationGroupInput, *StrategyConfig, error) {
+func (p *AWSRedisProvider) getRedisConfig(ctx context.Context, r *v1alpha1.Redis) (*elasticache.CreateReplicationGroupInput, *elasticache.DeleteReplicationGroupInput, *StrategyConfig, error) {
```

Comment:
Can we rename this function to `getElasticacheConfig`, It is something we have been currently doing on the refactor for the aws providers, we are using the aws resource name.